tensorflow_addons/examples/Normalizations.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<p>Can you try to match the `tensorflow/docs` template as much as possible:</p><p><a href="https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb" target="_blank">https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb</a></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>